translations/translations.json

```diff
@@ -313,6 +313,7 @@
   },
   "common": {
     "de": {
+      "moreOptions": "Weitere Optionen",
```

Comment:
Did you have a look what accessibility label other apps are setting? Not sure if this is a good one.

Reply:
Mattermost has "Mehr Optionen" if you prefer that :D

Reply:
Both is fine :) Mehr is perhaps a little bit simpler language but your choice :)